src/lib/translations.js

```diff
@@ -9,52 +9,52 @@ export default {
 		intro: {
 			title: 'Szanowna Użytkowniczko, Szanowny Użytkowniku,',
 			description: `<p> Zanim klikniesz „Przejdź do serwisu” lub zamkniesz to okno, prosimy o przeczytanie tej informacji. Prosimy w niej o Twoją dobrowolną zgodę na przetwarzanie Twoich danych osobowych przez naszych partnerów biznesowych oraz przekazujemy informacje o tzw. cookies i o przetwarzaniu przez nas Twoich danych osobowych. <span style="font-weight: bold">Klikając „Przejdź do serwisu” lub zamykając okno przez kliknięcie w znaczek X, zgadzasz się na poniższe.</span> Możesz też odmówić zgody lub ograniczyć jej zakres.</p>
-			
+
     <p style="margin: 25px 0 10px 0; font-weight: 900">Zgoda</p>
-	
-    <p> Jeśli chcesz zgodzić się na przetwarzanie przez <a onclick="__cmp('showVendors')">Zaufanych Partnerów</a> Grupy RAS Polska Twoich danych osobowych, które udostępniasz w historii przeglądania stron i aplikacji internetowych oraz danych lokalizacyjnych generowanych przez Twoje urządzenie, w celach marketingowych (obejmujących zautomatyzowaną analizę Twojej aktywności na stronach internetowych i w aplikacjach w celu ustalenia Twoich potencjalnych zainteresowań dla dostosowania reklamy i oferty) w tym na umieszczanie znaczników internetowych (cookies itp.) na Twoich urządzeniach i odczytywanie takich znaczników, kliknij przycisk „Przejdź do serwisu” lub zamknij to okno. </p>
-	
+
+    <p> Jeśli chcesz zgodzić się na przetwarzanie przez <a onclick="__tcfapi('showVendors', 2, () => {})">Zaufanych Partnerów</a> Grupy RAS Polska Twoich danych osobowych, które udostępniasz w historii przeglądania stron i aplikacji internetowych oraz danych lokalizacyjnych generowanych przez Twoje urządzenie, w celach marketingowych (obejmujących zautomatyzowaną analizę Twojej aktywności na stronach internetowych i w aplikacjach w celu ustalenia Twoich potencjalnych zainteresowań dla dostosowania reklamy i oferty) w tym na umieszczanie znaczników internetowych (cookies itp.) na Twoich urządzeniach i odczytywanie takich znaczników, kliknij przycisk „Przejdź do serwisu” lub zamknij to okno. </p>
```

Comment:
Please don't use arrow functions here because this code is not transpiled by Babel. Additionally as fas as I see `callback` argument is not required by "showVendors" command.

Reply:
Unfortunately every tcfapi call needs third parameter - callback. I've just changed it to regular function (not arrow func)